optax/_src/alias.py

```diff
@@ -177,6 +177,82 @@ def adadelta(
   )
 
 
+def adan(
+    learning_rate: base.ScalarOrSchedule,
+    b1: float = 0.98,
+    b2: float = 0.92,
+    b3: float = 0.99,
+    eps: float = 1e-8,
+    eps_root: float = 1e-8,
+    weight_decay: float = 0.0,
+    mask: Optional[Union[Any, Callable[[base.Params], Any]]] = None,
+) -> base.GradientTransformation:
+  """The ADAptive Nesterov momentum algorithm (Adan).
+
+  Adan first reformulates the vanilla Nesterov acceleration to develop a new
+  Nesterov momentum estimation (NME) method, which avoids the extra overhead of
+  computing gradient at the extrapolation point. Then Adan adopts NME to
+  estimate the gradient's first- and second-order moments in adaptive gradient
+  algorithms for convergence acceleration.
+
+  References:
+    Xie et al, 2022: https://arxiv.org/abs/2208.06677
+
+  Args:
+    learning_rate: this is a fixed global scaling factor.
+    b1: Decay rate for the exponentially weighted average of gradients.
+    b2: Decay rate for the exponentially weighted average of difference of
+      gradients.
+    b3: Decay rate for the exponentially weighted average of the squared term.
+    eps: a small constant applied to denominator outside of the square root
+      (as in the Adam paper) to avoid dividing by zero when rescaling.
+    eps_root: A small constant applied to denominator inside the square root (as
+      in RMSProp), to avoid dividing by zero when rescaling.
+    weight_decay: strength of the weight decay regularization.
+    mask: a tree with same structure as (or a prefix of) the params PyTree,
+      or a Callable that returns such a pytree given the params/updates.
+      The leaves should be booleans, `True` for leaves/subtrees you want to
+      apply the weight decay to, and `False` for those you want to skip. Note
+      that the Adam gradient transformations are applied to all parameters.
+
+  Returns:
+    the corresponding `GradientTransformation`.
```

Comment:
I think that ```:class:`optax.GradientTransformation` ``` should result in a nicer clickable link in the webpage. please build locally and check this.